Source/BoogieDriver/BoogieDriver-NetCore.csproj

```diff
@@ -9,12 +9,14 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <AssemblyName>BoogieDriver</AssemblyName>
+    <AssemblyName>Boogie</AssemblyName>
```

Comment:
Why this change? Maybe better to maintain the legacy structure as much as possible?

Reply:
I thought it's nice to call the "main" executable DLL just `Boogie.dll`, but I understand the desire to name it the same as the project. I'll rename it back to `BoogieDriver`.
But you agree with `Binaries/` in the repository root as the output path?

Reply:
Well, since that is how Boogie was built before the `-NetCore.csproj` variations, yes.